drafts/2016-01-06.md

```diff
@@ -0,0 +1,91 @@
+Welcome to the first edition of IPFS Weekly!  [IPFS](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open.  In these posts, we will try to highlight some of development that happened in the past week.  For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [github](https://github.com/ipfs) or join us on [IRC](http://webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+Since this is our first time launching the Weekly, we've included several past weeks. This is partially because we've been refining our process, and wanted to make the first weekly a great one. In the future, they will be released weekly. For now, here is a summary of what has happened over at IPFS for the past four weeks. If you have any feedback about this process in general, let us know [here](https://github.com/ipfs/weekly/issues/7). Thanks!
+
+## December 21
+
+Here are some of the highlights for the [December 21](https://github.com/ipfs/pm/issues/76) Sprint:
+
+Not too much happened during this sprint, because it was the holidays. [@jbenet](//github.com/jbenet) was off visiting Middle Earth (New Zealand), and [@whyrusleeping](//github.com/whyrusleeping), [@diasdavid](//github.com/diasdavid), [@lgierth](//github.com/lgierth), and more of the team were over at 32c3 in Hamburg.
```

Comment:
Do we know if it is ok to share that widely? Who would know? @whyrusleeping @Dignifiedquire